src/Stratis.SmartContracts.Executor.Reflection/InternalTransactionExecutor.cs

```diff
@@ -48,7 +50,8 @@ public ITransferResult TransferFunds(ISmartContractState smartContractState, Add
         {
             this.logger.LogTrace("({0}:{1},{2}:{3})", nameof(addressTo), addressTo, nameof(amountToTransfer), amountToTransfer);
 
-            // TODO: The act of calling this should cost a lot of gas!
+            // TODO: Spend BaseFee here
```

Comment:
Is this comment still relevant then?

Reply:
Yep! We need to spend the basefee as soon as we enter that part :)